samples/bootloader/README.rst

```diff
@@ -49,7 +49,8 @@ The NSIB implements a simple and reliable :term:`Root of Trust (RoT)` for a secu
 
    Each slot has a version number, and the bootloader will select the slot with the latest version.
 
-   For more information about creating a second stage bootloader, see :ref:`ug_bootloader_adding_upgradable`.
+   :ref:`Static configuration for defining static partitions <ug_pm_static>` must be used when using DFU.
```

Comment:
Should it actually belong here? The section seems to focus on how bootloader works, not on configuration

Reply:
Moved to Configuration.